### PR TITLE
Update modules to nf-core ( clustalo and famsa)

### PR DESCRIPTION
**Pull request by luisas**
Move modules of famsa and clustalo to nf-core modules. 
Update corresponding subworkflows.

### CODE REVIEW

File: conf/test.config [hunk 1]
```diff
@@ -19,6 +19,10 @@ params {
     max_memory = '6.GB'
     max_time   = '6.h'
 
+    // Stats
+    skip_stats                 = true
+    skip_eval                  = true
```
Comment (JoseEspinosa) [inline conf/test.config:24]:
```suggestion
    skip_stats = true
    skip_eval  = true 
```

File: assets/samplesheet.csv [hunk 2]
```diff
@@ -1,3 +1,3 @@
 id,fasta,reference,structures
-seatoxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/setoxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/setoxin.ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/structures/setoxin-ref
-toxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/toxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/toxin.ref,
+seatoxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/setoxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/setoxin.ref,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/structures/setoxin-ref
+toxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/toxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/toxin.ref,
```
Comment (maxulysse) [inline assets/samplesheet.csv:3]: Looks like local paths

File: conf/modules.config [hunk 3]
```diff
@@ -36,6 +36,16 @@ process {
         ]
     }
     
+    withName: "CLUSTALO_GUIDETREE"{
+        ext.prefix = { "${meta.id}_${meta.tree}-args-${meta.argstree_clean}" }
+        ext.args = { "${meta.args_tree}" == 'none' ? '' : "${meta.args_tree}" }
+        publishDir = [
```
Comment (JoseEspinosa) [inline conf/modules.config:42]:
```suggestion
        ext.prefix = { "${meta.id}_${meta.tree}-args-${meta.argstree_clean}" }
        ext.args   = { "${meta.args_tree}" == 'none' ? '' : "${meta.args_tree}" }
        publishDir = [
```

File: assets/samplesheet.csv (hunk 2 above)
Comment (luisas) [reply assets/samplesheet.csv]: Ups, yes! fixed it now :)

Comment (maxulysse) [reply assets/samplesheet.csv]: where?

Comment (maxulysse) [reply assets/samplesheet.csv]: nevermind, was looking at the wrong place

File: assets/samplesheet.csv [hunk 4]
```diff
@@ -1,3 +1,3 @@
 id,fasta,reference,structures
-seatoxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/setoxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/setoxin.ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/structures/setoxin-ref
-toxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/toxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/toxin.ref,
+seatoxin-ref,../test-dataset/setoxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/setoxin.ref,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/structures/setoxin-ref
+toxin-ref,../test-dataset/toxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/toxin.ref,
```
Comment (maxulysse) [inline assets/samplesheet.csv:3]: you still have some local paths leftovers

Comment (luisas) [reply assets/samplesheet.csv:3]: oh ok, yes i have those files in the repo. I imagine this is not supposed to happen then? Should i add it to the nf-core test datasets instead?

File: nextflow.config [hunk 5]
```diff
@@ -227,7 +227,7 @@ manifest {
     description     = """Pipeline to run and benchmark multiple sequence alignment tools."""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '1.0dev'
+    version = '0.0.1dev'
```
Comment (JoseEspinosa) [inline nextflow.config:230]:
```suggestion
    version = '0.1.0dev'
```

File: subworkflows/local/align.nf [hunk 6]
```diff
@@ -23,46 +25,98 @@ workflow ALIGN {
...
     ch_versions = ch_versions.mix(COMPUTE_TREES.out.versions)
 
+
+    // Separate the computation intothose which need a tree and those which don't
+    ch_fasta_tools = ch_fastas.combine(ch_tools).map{ it -> [it[0] + it[2] ,  it[3], it[1]] }
+                                                .branch{
+                                                    with_tree: it[0]["tree"] != "none"
+                                                    without_tree: it[0]["tree"] == "none"
+                                                }
+
```
Comment (JoseEspinosa) [inline subworkflows/local/align.nf:42]: For readability I will probably do it this way, but up to you:
```suggestion
    ch_fastas.combine(ch_tools)
        .map{ it -> [it[0] + it[2] ,  it[3], it[1]] }
        .branch {
            with_tree: it[0]["tree"] != "none"
            without_tree: it[0]["tree"] == "none"
        }
        .set { ch_fasta_tools }

```

File: subworkflows/local/align.nf [hunk 7]
```diff
@@ -23,46 +25,98 @@ workflow ALIGN {
...
-                              .map{ it -> [it[0] + it[2] ,  it[3], it[1]] }
+    ch_fasta_trees = ch_fasta_tools.with_tree
                               .combine(trees, by: [0])
                               .map{ it -> [it[0] + it[1] , it[2], it[3]]}
                               .branch{
                                   famsa: it[0]["align"] == "FAMSA"
                                   tcoffee3D_tmalign: it[0]["align"] == "tcoffee3D_tmalign"
                                   tcoffee_regressive: it[0]["align"] == "regressive"
+                                  clustalo: it[0]["align"] == "CLUSTALO"
                               }
```
Comment (JoseEspinosa) [inline subworkflows/local/align.nf:53]:
```suggestion
    ch_fasta_tools
        .with_tree
        .combine(trees, by: [0])
        .map { it -> [it[0] + it[1] , it[2], it[3]]} 
        .branch {
            famsa: it[0]["align"] == "FAMSA"
            tcoffee3D_tmalign: it[0]["align"] == "tcoffee3D_tmalign"
            tcoffee_regressive: it[0]["align"] == "regressive"
            clustalo: it[0]["align"] == "CLUSTALO"
        }
        .set { ch_fasta_trees }
```

File: subworkflows/local/align.nf [hunk 8]
```diff
@@ -23,46 +25,98 @@ workflow ALIGN {
...
 
     //    
     // Compute the alignments
     // 
-    // FAMSA 
-    FAMSA_ALIGN(ch_fasta_trees.famsa)
+
+    // -----------------   FAMSA ---------------------
+    ch_fasta_trees_famsa = ch_fasta_trees.famsa.multiMap{
```
Comment (JoseEspinosa) [inline subworkflows/local/align.nf:60]: same here and below in similar instances

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Just some small comments to be addressed.
Also, local modules will need to have their own container even if they are running python scripts so this might need to be updated in a new PR also the test data should not be local, but this is something that also could be solve in a different PR.

**Review by JoseEspinosa — APPROVED** [review]
LGTM! Let's merge this one and work in the remaining fixes in follow up PRs
Will also close #49

File: LICENSE [hunk 9]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) Luisa Santus, Jose Espinosa Carrasco
+Copyright (c) luisas
```
[the remark this answers is not in the document]
Comment (luisas) [reply LICENSE]: Ok perfect!

**Pull request merged**